site/app/controllers/PollController.php

```diff
@@ -152,7 +154,14 @@ public function addNewPoll() {
                 $answers[] = $_POST["option_id_" . $i];
             }
         }
-        $this->core->getQueries()->addNewPoll($_POST["name"], $_POST["question"], $responses, $answers, $_POST["release_date"], $orders);
+        $poll_id = $this->core->getQueries()->addNewPoll($_POST["name"], $_POST["question"], $responses, $answers, $_POST["release_date"], $orders);
+        $file_path = null;
+        if (array_key_exists("image_file", $_FILES) && $_FILES["image_file"]["name"] != "") {
```

Comment:
Probably safer to use `isset` here as if `$_FILES['image_file] === null`, then this if statement would pass. `isset` passes only if the key exists and its value is not null, plus it's less characters.

```suggestion
        if (isset($_FILES['image_file']) && $_FILES["image_file"]["name"] !== "") {
```